task/buildah-oci-ta/0.4/buildah-oci-ta.yaml

```diff
@@ -398,6 +405,17 @@ spec:
           esac
         done
 
+        # Append any additional annotations from the specified file
```

Comment:
IMO `ANNOTATIONS` should take precedence over `ANNOTATIONS_FILE` the same way that `BUILD_ARGS` take precedence over `BUILD_ARGS_FILE`. We would probably achieve that by parsing the annotations file first and then appending the regular annotations?

Reply:
Like this? -> https://github.com/konflux-ci/build-definitions/pull/2207/files#diff-8ececcbe0dae0061a00d374353e73b4c75b606c645b90e1546ea1dcff69bb748R314-R349

Reply:
Yup, that should work :+1: